udata/core/user/views.py

```diff
@@ -3,8 +3,9 @@
 
 import logging
 
-from flask import g
+from flask import abort, g
 
+from udata.core.user.api import is_not_available
```

Comment:
No, there shouldn't be a dependency from `views` to `api`, so it might either be a `user.models.User` method or `user.models` function.
**Proposal**: a `is_visible()` method on `User` model